spec/Machine/sample-10-spec.js

```diff
@@ -19,8 +19,8 @@ describe('Machine', () => {
   });
 
   it('should do something alright #1', () => {
-    expect(machine.execute('@include "input.nut"')).toBe(result);
+    expect(machine.execute('@include "input.nut"').replace(/\\/g, '/')).toBe(result);
     machine.generateLineControlStatements = true;
-    expect(machine.execute('@include "input.nut"')).toBe(resultWithLC);
+    expect(machine.execute('@include "input.nut"').replace(/\\/g, '/')).toBe(resultWithLC);
```

Comment:
I'd prefer not to have this unobvious code. Is this the only test for windows path? May we generate resultWithLC?